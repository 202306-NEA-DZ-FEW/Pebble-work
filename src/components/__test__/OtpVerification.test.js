import React from "react";
import renderer from "react-test-renderer";

import OtpInput from "../OtpVerification";

test("OtpInput snapshot", () => {
    const component = renderer.create(<OtpInput />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
