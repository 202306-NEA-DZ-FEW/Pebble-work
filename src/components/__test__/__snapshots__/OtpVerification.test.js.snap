// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OtpInput snapshot 1`] = `
<div
  className="otpContainer"
>
  <input
    autoFocus={true}
    maxLength={1}
    onChange={[Function]}
    onKeyDown={[Function]}
    type="text"
    value=""
  />
</div>
`;
